.github/workflows/test.yml

```diff
@@ -13,9 +13,17 @@ jobs:
     runs-on: ubuntu-latest
     name: Lint and Test
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+    - uses: shogo82148/actions-setup-mysql@v1
```

Comment:
This action installs MariaDB to be able to use WP_Unit for the API tests. Making use of WP_Unit here means we don't need to dance around a lot of API-related file inclusions in the bootstrap file anymore.